scripts/after-pack-hook.js

```diff
@@ -99,7 +118,31 @@ module.exports = async function (params) {
       await cleanupUnneededDependencies(outputFolder)
       await setupV8Snapshots({
         cypressAppPath: params.appOutDir,
-        integrityCheckSource: getIntegrityCheckSource(outputFolder),
+        integrityCheckSource,
+      })
+
+      await createJscFromCypress()
+
+      const indexJscHash = await getIndexJscHash(outputFolder)
+
+      await fs.remove(path.join(outputFolder, 'packages/server/index.js'))
```

Comment:
```suggestion
      // This file is not needed at this point since it's been replaced by the jsc. So we remove it.
      await fs.remove(path.join(outputFolder, 'packages/server/index.js'))
```